scss/main.scss

```diff
@@ -102,3 +102,5 @@ nav {
     }
   }
 }
+
+@import 'responsive';
```

Comment:
I think it's better to put it on the top of the document, where all other `@import` statements are, don't you think?